DailyReportWeb/Helpers/PolicyService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using SourceControlLogReporter.Model;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Web;
+
+namespace DailyReportWeb.Helpers
+{
+    public class PolicyService
+    {
+        public static void SetPolicyFullDraft(Policy policy, string policyPath)
+        {
+            policy.AdvancedOptions.NoIndividualDraft = true;
+            policy.AdvancedOptions.NoDraft = false;
+            policy.GeneratedProperties.IndividualDrafts = null;
+            policy.SaveToFile(policyPath);
+        }
+
+        public static void SetPolicyFinalReport(Policy policy, string policyPath)
+        {
+            policy.AdvancedOptions.NoDraft = true;
```

Comment:
@DanBadea 
Pai trebuie setate astea ca urmatoarea data cand se trimite raportul, sa fie clar de care raport e: individual, draft sau final. 
As putea sa pun proprietati noi in GeneratedProperties. Dar raportul se seteaza in functie de tipul specificat in AdvancedOptions. SI de asta cred ca trebuie modificate sa se stie ce tip de raport trebuie trimis.